NEWS.md

```diff
@@ -1,6 +1,15 @@
 ## 23.1.0 IN-PROGRESS
 
 * Add related instances record API (MODINVSTOR-861)
+* Added integrity checks to statisticalCodeIds in instance records (MODINVSTOR-885)
```

Comment:
I think the removal of the UUID constraint on statistical codes should be included in this list